### PR TITLE
Update for 1.18.1

### LINKED ISSUE #151: [Feature] Add Trinkets support for backpacks
Add support for backpacks to be put in the [Trinkets](https://www.curseforge.com/minecraft/mc-mods/trinkets-fabric) `chest/back` slot without dropping contents, and still be accessible via hotkey as per usual. When this option is turned on, prevent the backpack from being equipped in the usual chest slot.

Balance implications of proposed change:
* Satisfies internal logic added by Trinkets ("makes sense" to go in the backpack slot)
* Prevents wearing a second backpack trinket by filling the appropriate slot; otherwise you can equip a Campanion backpack and another backpack.
* Allows wearing armor with backpack _but_ prevents potentially wearing a buff-adding Trinket - this is probably a net buff due to customizable armor enchantments and mod-added armor sometimes being *quite* powerful, whereas there haven't been as "OP" trinkets in the community so far.

My recommendation would be to make this support configurable, defaulting to off to maintain current game balance; mod-pack curators and server admins can turn it on where desired.

Of course, I'd be turning it on immediately because it feels weird not to be able to put it in the back slot.

### LINKED ISSUE #155: I did some Item re textures and a Brazilian Portuguese translation
Hey guys. I did some **re textures** for my personal playthrough and decided to share with you. I'm leaving attached a .zip folder with all .pngs if you want to use them in the original mod.

And I did a **PT-BR (Brazilian Portuguese)** translation file too, I'll let it attached as well.

Hope you like them!

It's been very fun to play with your mod, guys, congratulations for the excellent job!

Showcase

Showcase - original sprite sizes

[Campanion Item Retextures.zip](https://github.com/TerraformersMC/Campanion/files/7512317/Campanion.Item.Retextures.zip)

[pt_br.zip](https://github.com/TerraformersMC/Campanion/files/7512314/pt_br.zip)

### LINKED ISSUE #156: Tent blocks should be immune to explosions (while not shielding the contents)
Currently tents are irreversibly damaged when an explosion hits them. This affects function other than looks, because the removed blocks give more space to place more stuff, but of course it doesn't look like a tent anymore.

So maybe the blocks should be immune to explosions. For balance they shouldn't be able to shield what's inside from explosions either, if possible.

### PR DESCRIPTION
**Pull request by jhbuchanan45**
These changes should enable the mod to run properly on 1.18.1.

In addition:
 - Closes #151: Added a config option to allow backpacks to be placed into trinkets mod 'back' slots, disabled by default
 - Closes #156: All blocks making up tents have the blast resistance of obsidian now
 - Closes #155: Adds new item textures and Brazilian Portuguese translations, thanks to @ppblitto

### CODE REVIEW

File: src/main/java/com/terraformersmc/campanion/block/CampanionBlocks.java [hunk 1]
```diff
@@ -113,7 +113,7 @@
 	public static final TentTopFlatBlock RED_FLAT_TENT_TOP = tentTopFlat(DyeColor.RED);
 	public static final TentTopFlatBlock BLACK_FLAT_TENT_TOP = tentTopFlat(DyeColor.BLACK);
 
-	public static final Block TENT_POLE = add("tent_pole", new TentPoleBlock(FabricBlockSettings.of(Material.WOOD).nonOpaque().hardness(-1F).sounds(BlockSoundGroup.WOOD)), (ItemGroup) null);
+	public static final Block TENT_POLE = add("tent_pole", new TentPoleBlock(FabricBlockSettings.of(Material.WOOD).nonOpaque().hardness(-1F).resistance(1200F).sounds(BlockSoundGroup.WOOD)), (ItemGroup) null);
```
Comment (NeusFear) [inline src/main/java/com/terraformersmc/campanion/block/CampanionBlocks.java:116]: Not sure if this is the right option to go with here

**Review by NeusFear — COMMENTED** [review]
Everything port-wise looks good. Still not sure about the explosion-resistant property of the tent block though,

File: src/main/java/com/terraformersmc/campanion/block/CampanionBlocks.java (hunk 1 above)
Comment (NeusFear) [reply src/main/java/com/terraformersmc/campanion/block/CampanionBlocks.java:116]: After discussion on discord this should be fine for now.

**Review by NeusFear — APPROVED** [review]

**Review by Wyn-Price — APPROVED** [review]

**Comment by Wyn-Price** [comment]
I haven't tested it yet, but I trust that it all works

**Pull request merged**